<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JumpEventPropertiesClass</class>
 <widget class="QFrame" name="JumpEventPropertiesClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>80</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JumpEventProperties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="CollapseButton" name="jumpEventCollapseButton">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextBesideIcon</enum>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
     <property name="arrowType">
      <enum>Qt::RightArrow</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="jumpEventContent">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="1">
       <widget class="QLineEdit" name="jumpToSceneLineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="conditionLineEdit"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="jumpToSceneLabel">
        <property name="text">
         <string>Jump to scene name:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="conditionLabel">
        <property name="text">
         <string>Condition:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CollapseButton</class>
   <extends>QToolButton</extends>
   <header>CollapseButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
